src/frontend/data/appData.ts

```diff
@@ -9,12 +9,26 @@ export const appData = parseDataElements(
 
```

Comment:
Just a typo in commit message:
❌ `... definition. This type return ...`
✅ `... definition. This type returns ...`